src/ocamlorg_frontend/pages/releases.eml

```diff
@@ -13,69 +13,74 @@ Layout.render
 </div>
 <div class="bg-default dark:bg-dark-default">
     <div class="py-10 lg:py-28">
-        <div class="container-fluid">
+        <div class="container-fluid min-w-full">
```

Comment:
I would go for `container mx-auto`  instead, since container fluid wasn't a great choice here in the original design. `container` is more predictable in how it resizes. (Avoiding the fluid container here means we just have a few breakpoints to check regarding whether everything looks right.)